modules/rostests/apitests/user32/PrivateExtractIcons.c

```diff
@@ -3,41 +3,128 @@
  * LICENSE:         LGPLv2.1+ - See COPYING.LIB in the top level directory
  * PURPOSE:         Test for PrivateExtractIcons
  * PROGRAMMER:      Hermes Belusca-Maito
+ *                  Doug Lyons <douglyons@douglyons.com>
  */
 
 #include "precomp.h"
+#include <stdio.h>
+
+BOOL FileExists(LPCSTR FileName)
+{
+    FILE *fp = NULL;
+    BOOL exists = FALSE;
```

Comment:
This variable looks superfluous: simply use `return FALSE/TRUE`.

Reply:
Revised to remove the BOOL variable. Thanks.